src/_P121_HMC5883L.ino

```diff
@@ -127,7 +127,7 @@ boolean Plugin_121(uint8_t function, struct EventStruct *event, String &string)
 
       float heading = atan2(s_event.magnetic.y, s_event.magnetic.x);
 
-      const float decl = PCONFIG(0);
+      const float decl = PCONFIG_FLOAT(0);
       addLog(LOG_LEVEL_DEBUG, "Declination Degree:" + String(decl));
```

Comment:
Ah, another candidate for the `F()` macro, and checking for DEBUG log, like
```
# ifndef BUILD_NO_DEBUG
if (isLogLevelActiveFor(LOG_LEVEL_DEBUG) {
  String log = F("Declination Degree: ");
  log += decl;
  addLog(LOG_LEVEL_DEBUG, log);
}
# endif // ifndef BUILD_NO_DEBUG
```

I assume there will be a few more instances similar to this 😉

Reply:
I just removed it. I added it, because it stayed 0.00 all the time. Now it is working, therfore I removed the debug message.